pkg/kindsys/kindcats.cue

```diff
@@ -81,7 +81,7 @@ _sharedKind: {
 
 // Maturity indicates the how far a given kind declaration is in its initial
 // journey. Mature kinds still evolve, but with guarantees about compatibility.
-#Maturity: "merged" | "experimental" | "stable" | "mature"
+Maturity: "merged" | "experimental" | "stable" | "mature"
```

Comment:
Jooc, what's the goal, or what are the actual implications of turning `Maturity` from a definition into a value? 🤔

Reply:
It has no real effect here. But it's primarily about consistency with the need to not use definitions elsewhere (e.g. i have to change ~all of Thema's base cue types), because of how they recursively close anything within them. That creates a bunch of problems when the things inside a definition are arbitrary schema - such as are contained within the `Core`, `Custom`, and `Composable` kind defs in kindsys, here. So, getting away from that is the _desired_ effect of not using definitions.

Unfortunately, IMO CUE kinda overloads definitions with a bunch of things - they're not included in default output from e.g. `cue eval`, and many of its encoders/generators treat being a definition as indicative that a thing should be generated. i don't know what all else we might run into with it, but the recursive closing issue is so problematic for schema definitions that are useful outside of CUE that we're just going to have to deal with whatever come sup.